Sources/TensorFlow/BackwardsCompatibility.swift

```diff
@@ -146,3 +146,55 @@ public func sigmoidCrossEntropy<Scalar: TensorFlowFloatingPoint>(
 ) -> Tensor<Scalar> {
     sigmoidCrossEntropy(logits: logits, labels:labels, reduction: _mean)
 }
+
+public extension Tensor {
+    /// Returns a transposed tensor, with dimensions permuted in the specified order.
+    @inlinable
+    @differentiable(
+            wrt: self,
```

Comment:
Indent by 4. Same for other occurrences.
```suggestion
        wrt: self,
```

Reply:
Very good. I will work on the PR that introduces the renaming. Thanks for the feedback.